cranelift/codegen/src/isa/aarch64/inst/args.rs

```diff
@@ -645,3 +648,30 @@ impl VectorSize {
         }
     }
 }
+
+//=============================================================================
+// Instruction sub-components: atomic memory update operations
+
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+#[repr(u8)]
+pub enum AtomicRMWOp {
+    Add,
+    Sub,
+    And,
+    Or,
+    Xor,
+    Xchg,
+}
+
+impl AtomicRMWOp {
```

Comment:
Tiny nit, but this was very confusing to me for a moment; the difference between the CLIF's `AtomicRmwOp` and the aarch64 `Inst`'s `AtomicRMWOp` is just capitalization. Perhaps `use crate::ir` and then write `ir::AtomicRmwOp` here, for clarity?

Reply:
Er, well spotted.  I didn't even notice.  Must have looked as if I'd simply written the identity function for atomic RMW ops :-)